rct-graphics-helper/properties/general_properties.py

```diff
@@ -94,6 +94,14 @@ class GeneralProperties(bpy.types.PropertyGroup):
         description="Which color groups to dither to. Recolorables will be excluded from this palette when used to avoid conflicts.",
         size=len(defaults))
 
+    dithering_strength = bpy.props.IntProperty(
+        name = "Dithering Strength",
```

Comment:
Rather than strength perhaps it should be kept to its actual name "Dithering Error Diffusion"

Reply:
"Strength" is a common term used by image modifiers in image editing programs. Photoshop just calls it Dither %. Strength is concise and describes the effect the property has: weak dither has more shelving, strong dither has more noise.